metrics/src/init.ts

```diff
@@ -0,0 +1,47 @@
+import { DiagConsoleLogger, DiagLogLevel, diag, metrics } from '@opentelemetry/api';
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http';
+import { OTLPMetricExporter as OTLPMetricExporterGRPC } from '@opentelemetry/exporter-metrics-otlp-grpc';
+import { MeterProvider, PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics';
+import { Resource } from '@opentelemetry/resources';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+import packageInfo from '../package.json';
```

Comment:
will this work for an npm?

Reply:
Made it passable from outside https://github.com/boxyhq/metrics/blob/main/src/init.ts#L16